<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 94, 94);</string>
   </property>
   <layout class="QGridLayout" name="centralGrid">
    <property name="spacing">
     <number>15</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSuper">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuDiffilculty">
     <property name="title">
      <string>Diffilculty</string>
     </property>
     <addaction name="actionEasy"/>
     <addaction name="actionMedium"/>
     <addaction name="actionHard"/>
     <addaction name="actionExtreme"/>
     <addaction name="actionUnwinnable"/>
    </widget>
    <addaction name="menuDiffilculty"/>
    <addaction name="actionPVP"/>
   </widget>
   <addaction name="menuSuper"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPVP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PVP</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionExtreme">
   <property name="text">
    <string>Extreme</string>
   </property>
  </action>
  <action name="actionUnwinnable">
   <property name="text">
    <string>Unwinnable</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
